packaging/fedora/snapd.spec

```diff
@@ -47,6 +47,23 @@
 
 %global snappy_svcs     snapd.service snapd.socket snapd.autoimport.service snapd.refresh.timer snapd.refresh.service
 
+# Until we have a way to add more extldflags to gobuild macro...
+%if 0%{?fedora} >= 26
+%define gobuild_static(o:) go build -buildmode pie -compiler gc -tags=rpm_crashtraceback -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '%__global_ldflags -static'" -a -v -x %{?**};
```

Comment:
Really like enabling `-v -x` 👍 Invaluable when hunting down the 'weird' stuff.

Reply:
This is actually part of the default flags for `%gobuild` in Fedora. :)

Reply:
o_O not even sure what's going but I'm in awe for that non-trivial line.